### ISSUE TITLE
Wire & Rocket.Chat

### ISSUE THREAD

**Issue by cryptono** [nylira/prism-break#1514]
Please add:

https://www.wire.com
Open Source, encrypted messenger based in Switzerland

https://rocket.chat
Open Source, with OTR encrypted chat

Edited:
https://www.ghostmail.com
Open Source E-Mail-, Chat- and Storage service

**Comment by mastercoms** [comment]
Wire's client is not open source.

**Comment by cryptono** [comment]
@mastercoms This is a point, but they already answered this critic:
"Our first priority was to open source the parts related to cryptography as well as the data handling part.
We will soon add more documentation and build instructions. We would also like to open source more parts of the application in the future.

In the meantime we will have a review of all our source code done by an independent third party, to provide the users with more guarantees about the parts that are not yet open sourced."
And: they are very new, there'll be a Linux client probably soon.

As they seem reliable to me (https://wire.com/privacy/), I would give them a chance.

**Comment by mastercoms** [comment]
Could you link me to this quote?

They are not very new. They've been around since late 2014. I have asked them before about a Linux client. Twice. One before they released Wire 2.0, and one after.

Before Wire 2.0:
> Thank you for contacting Wire to share your feedback. Not at this stage, but we are always looking for ways to improve our product and we appreciate your taking the time to send us your comments.

After Wire 2.0:
> thank you for writing to Wire.
We take your feedback very much into account and we will discuss it with our teams.
I know you want a native app, right now Wire for Web can help.
I'll forward your ideas to our dev team.
Unfortunately we can not say if and when this feature comes out. We will announce all releases through our social media channels, blog and website.

It does not matter if you trust it or if it will be open sourced in the future. Prism Break only allows open source projects. Look, I use Wire myself for a long time now, but it can't be included on Prism Break. Not until these issues are fixed.

**Comment by cryptono** [comment]
@mastercoms 
https://github.com/wireapp/zmessaging-android/issues/1
You are right. I hope they'll publish the missing code soon...I don't know any better messaging alternative then Wire. Do you?

Nevertheless: Rocket.Chat is Open Source and hostable for any person.

**Comment by merlinnusr** [comment]
Rocket.chat don't have  otr support they will adding
Ghost mail it's a good alternative to riseup(Hosted in USA ) I don't know i admin don't ban Riseup.

**Comment by cryptono** [comment]
@merlinnusr Rocket.Chat supports OTR since v0.22.0. Play it safe: Use Rocket.Chat or read the release descriptions: https://github.com/RocketChat/Rocket.Chat/releases

**Comment by Zegnat** [comment]
1. Wire does not seem to be usable without running proprietary software, as @mastercoms points out, and will not be included here.
   
2. Rocket.Chat indeed warrants some more research, and could also be listed as a Slack replacement on PRISM Break. (People seem to want that sort of thing.) If anyone wants to look into them and open a new issue that would be swell! I currently do not have the time for that.
   
3. GhostMail does not seem to be doing anything revolutionary. Their chat system looks to be using [Converse.js](https://conversejs.org) and is just XMPP in the browser.
   
   GhostMail then wrote their own cryptography wrapper ([KRYPTOS](https://github.com/ghostmail/GhostMailWeb/blob/master/js/kryptos/kryptos.core.js)) which seems to depend on both the Microsoft Research JavaScript Cryptography Library (of which they use an outdated version) and the Stanford Javascript Crypto Library (where they use a minified version without proper licence information making it hard for me to check the version number).
   
   And why did they do that? Who knows! Converse.js has integrated [OTR support](https://conversejs.org/docs/html/features.html#off-the-record-encryption) out of the box.

   You are better off using any one of the OTR supporting XMPP clients out there (and already on PRISM Break).

Closing.
